src/Microsoft.Restier.WebApi/Formatter/Deserialization/DefaultRestierDeserializerProvider.cs

```diff
@@ -11,11 +11,24 @@
 namespace Microsoft.Restier.WebApi.Formatter.Deserialization
 {
     /// <summary>
-    /// The default serializer provider.
+    /// The default deserializer provider.
     /// </summary>
-    internal class DefaultRestierDeserializerProvider : DefaultODataDeserializerProvider
+    public class DefaultRestierDeserializerProvider : DefaultODataDeserializerProvider
     {
         private RestierEnumDeserializer enumDeserializer;
+        private static readonly Lazy<DefaultRestierDeserializerProvider> lazy
```

Comment:
not necessary to use `Lazy`. Static instance will be created and initialized the first time it's being used.